docs/usage/guides/visual/system-tray.md

```diff
@@ -12,17 +12,20 @@ Configure the `systemTray` object on `tauri.conf.json`:
 {
   "tauri": {
     "systemTray": {
-      "iconPath": "icons/icon.png",
+      "iconPath": "icons/tray.png",
       "iconAsTemplate": true
     }
   }
 }
 ```
 
-The `iconPath` is pointed to a PNG file on macOS and Linux, and a `.ico` file must exist for Windows support.
```

Comment:
I don't see why this needs to be moved into another section, the wording could be better though.

```suggestion
- The `iconPath` is a path to a PNG file which will be used on macOS and Linux. You also have to include a `.ico` with the same name for Windows and Tauri will automatically detect it.
```